zkvm/src/predicate.rs

```diff
@@ -173,17 +186,19 @@ mod tests {
     #[test]
     fn valid_program_commitment() {
         let prog = Program::build(|p| p.drop());
-        let pred = Predicate::Program(prog.clone());
-        let op = pred.prove_program_predicate(&bytecode(&prog));
+        let pred = Predicate::unblinded_program(prog.clone());
+        let salt: [u8; 0] = [];
+        let op = pred.prove_program_predicate(&bytecode(&prog), &salt);
```

Comment:
maybe we can avoid using any extra variable?
```suggestion
        let op = pred.prove_program_predicate(&bytecode(&prog), &[]);
```